static_src/components/user_list.jsx

```diff
@@ -129,13 +129,11 @@ export default class UserList extends React.Component {
   }
 
   render() {
-    let saving;
+    let savingNotice;
```

Comment:
I would probably just omit the variable all together and put the `Loading` component directly into the JSX saved in `content`. But if we like using vars, go for it!